### DISCUSSION TITLE
How to make /opt/homeassistant/config/home-assistant_v2.db readable and writable for Sqlitebrowser

### DISCUSSION THREAD

**Discussion by Allinijhof** [sqlitebrowser/sqlitebrowser#3544] [Q&A] [answered]
Dear community,

I have just installed Sqlitebrowser through docker compose (see screenshot docker-compose.yaml file attached), unfortunately I am not able to access the home-assistant_v2.db file through Sqlitebrowser.

Would you be able to advise how to enable the folder and file for Sqlitebrowser?

With kind regards,

Allard
![Screenshot_2024-01-03-23-32-54-66_61c78dc80ee02b53007c815fefe993e3](https://github.com/sqlitebrowser/sqlitebrowser/assets/33557920/b9eb20e7-016f-4a8b-9bab-39b76d931f37)

**Comment by justinclift** [comment]
Interesting.  At a first guess, it's *probably* something like a user permissions problem.

You have the volume set to a persistent directory (`/opt/homeassistant/config`), which looks ok.

So the next thing you need to check is the ownership and permissions of that directory on your system,
then check if the user you are trying to access it from has permissions to get there.

Does that make sense? :smile:

Also, how good are you with using the command line / terminal?  If you're decently ok with it, that'll make it easier to give you potential commands to try out. :smile:

**Comment by FriedrichFroebel** [comment, marked as answer]
The Docker image is not maintained by DB4S itself, but by a third-party. It might be better to sort this out with the actual maintainers: https://github.com/linuxserver/docker-sqlitebrowser

**Reply by justinclift** [reply]
Good idea.  Looking at that page it has instructions for what looks like fixing this problem:

https://github.com/linuxserver/docker-sqlitebrowser#user--group-identifiers

**Comment by Allinijhof** [comment]
Hi Justin, Friedrich, thanks for your support! I have been able to direct the SQL to the right folder (volume). That is great thanks!

**Reply by justinclift** [reply]
Cool, sounds like a good result. :smile:

**Comment by Allinijhof** [comment]
Issue has been resolved.
